package.json

```diff
@@ -104,20 +84,35 @@
     "validator": "^9.1.2"
   },
   "devDependencies": {
+    "babel-cli": "^6.18.0",
```

Comment:
I shifted a bunch of build-time dependencies to `devDependencies`. This will allow us to cut down on the size of our docker image. I've done my best to move over everything that's only needed for build and nothing that would be needed at runtime.